### DISCUSSION TITLE
android build apk failed , please help

### DISCUSSION THREAD

**Discussion by EsmailDev21** [obytes/react-native-template-obytes#273] [Q&A]
Hello , I am trying to build a testing apk but I am encountering an error that  I didn't understand
C:\Users\Garai\Downloads\react-native-template-obytes-master>npm run build:staging:android

> obytesapp@4.3.2 build:staging:android
> cross-env APP_ENV=staging EXPO_NO_DOTENV=1 eas build --profile staging --platform android

★ eas-cli@7.2.0 is now available.
To upgrade, run npm install -g eas-cli.
Proceeding with outdated version.

Loaded "env" configuration for the "staging" profile: APP_ENV, EXPO_NO_DOTENV, FLIPPER_DISABLE. Learn more: https://docs.expo.dev/build-reference/variables/
You don't have the required permissions to perform this operation.

Entity not authorized: AccountEntity[e15599f8-a6e8-4fa4-8099-a4606ccda71e] (viewer = RegularUserViewerContext[57715e26-e030-4158-8b39-882baad5cfc2], action = READ, ruleIndex = -1)
Request ID: 4c29f40a-514d-419b-8516-f420fbbd9c60
    Error: GraphQL request failed.

**Comment by hamzamekk** [comment]
https://github.com/expo/expo-github-action/issues/228#issuecomment-1822401042 can you check this answer?

**Comment by EsmailDev21** [comment]
thank you , I will test this out
